extension/commercial_data/openapi.yaml

```diff
@@ -0,0 +1,328 @@
+openapi: 3.0.2
+info:
+  title: openEO Commercial data API extension
+  version: 0.1.0
+  description: 
+    $ref: README.md
```

Comment:
This is unfortunately invalid although redoc handles it quite nicely. The linter and other tools don't like it because it is not allowed in the OpenAPI spec and they expect an OpenAPI doc and not a Markdown file.

I'd propose to change to
```yaml
openapi: 3.0.2
info:
  title: openEO Commercial data API extension
  version: 0.1.0
  description: >-
    The Commercial Data API extension provides an interface for discovering,
    ordering and using commercial data in the openEO API.
  contact:
    name: openEO Consortium
    url: 'https://openeo.org'
    email: openeo.psc@uni-muenster.de
  license:
    name: Apache 2.0
    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
externalDocs:
  url: README.md
tags:
  - name: Orders
    description: Management of commercial data orders
```